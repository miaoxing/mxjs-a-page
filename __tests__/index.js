import React from 'react';
import {Page} from '..';
import {render} from '@testing-library/react';

describe('Page', () => {
  test('basic', () => {
    const result = render(<Page/>);
    expect(result.container).toMatchSnapshot();
  });
});

