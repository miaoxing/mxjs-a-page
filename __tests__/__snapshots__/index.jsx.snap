// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page basic 1`] = `
<div>
  <div>
    <nav
      class="ant-breadcrumb min-h-[22px] mb-4"
    >
      <ol />
    </nav>
  </div>
</div>
`;

exports[`Page breadcrumb match menu 1`] = `
<div>
  <div>
    <nav
      class="ant-breadcrumb min-h-[22px] mb-4"
    >
      <ol>
        <li>
          <a
            href="/admin"
          >
            首页
          </a>
        </li>
        <li
          aria-hidden="true"
          class="ant-breadcrumb-separator"
        >
          /
        </li>
        <li>
          <a
            href="/admin/test"
          >
            Test Page
          </a>
        </li>
        <li
          aria-hidden="true"
          class="ant-breadcrumb-separator"
        >
          /
        </li>
        <li>
          Child Page
        </li>
      </ol>
    </nav>
  </div>
</div>
`;

exports[`Page page with css props 1`] = `
<div>
  <div
    mb="2"
  >
    <nav
      class="ant-breadcrumb min-h-[22px] mb-4"
    >
      <ol />
    </nav>
  </div>
</div>
`;
